app/models/rent.rb

```diff
@@ -275,6 +227,14 @@ def as_json(*)
   end
 
   private
+
+  def renter_attributes
+    {
+        name: name, lastname: lastname, email: email,
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.